cmd/root.go

```diff
@@ -119,23 +122,43 @@ func rootCmd(o *options.Options) error {
 		return fmt.Errorf("GetEnabled: %w", err)
 	}
 
-	if o.Format == options.FormatDefault {
+	enabledProbes := o.Probes()
+	if len(enabledProbes) > 0 {
+		for _, probeName := range enabledProbes {
+			fmt.Fprintf(os.Stderr, "Starting probe [%s]\n", probeName)
+		}
```

Comment:
I'd prefer to see this in the same FormatDefault block below

Reply:
Done in https://github.com/ossf/scorecard/pull/3688/commits/7b40c188917c5526a174db3f1e4a2076bc15accc